### ISSUE TITLE
ResNet50RetinaNet - Pascal VOC.ipynb not in sync with current codebase

### ISSUE THREAD

**Issue by gat3way** [delftrobotics/keras-retinanet#11]
The evaluation part in the example isn't correct as of the current codebase. This part:

> boxes, classification, reg_loss, cls_loss = model.predict_on_batch(inputs)
    predicted_labels = np.argmax(classification[0, ...], axis=1)
    scores = np.max(classification[0, ...], axis=1)
    for idx, (label, score) in enumerate(zip(predicted_labels, scores)):
         b = boxes[0, idx, :].astype(int)
         cv2.rectangle(im, (b[0], b[1]), (b[2], b[3]), (0, 255, 0), 3)

    
should be more like this now:

> regression, boxes = model.predict_on_batch(inputs)
    for idx in range(0,boxes.shape[1]):
        label = np.argmax(boxes[0, idx,4:], axis=0)
        score = np.max(boxes[0, idx,4:], axis=0)

**Comment by hgaiser** [comment]
Correct! I'll update the ipynb soon and add one for COCO too.
